src/panels/config/script/ha-script-editor.ts

```diff
@@ -99,72 +96,7 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
 
   @state() private _validationErrors?: (string | TemplateResult)[];
 
-  private _schema = memoizeOne(
-    (
-      hasID: boolean,
-      useBluePrint?: boolean,
-      currentMode?: (typeof MODES)[number]
-    ) =>
-      [
-        {
-          name: "alias",
-          selector: {
-            text: {
-              type: "text",
-            },
-          },
-        },
-        {
-          name: "icon",
```

Comment:
Can we drop `icon` from the UI? It's not clear to me how this interacts with icon from the entity registry, if one is considered priority over the other, or if this is just vestigial. 

If we want to keep it I guess could fork the rename dialog (or share it with automations) and add it there.

Reply:
I think we should drop it to be aligned with automation editor.
Currently, automation editor save name (as `alias` key) and description to yaml and icon to entity registry. You can also override the name in the entity registry.

IMO, `name`, `description` and `icon` should be editable without opening the more info settings dialog.
`name` and `icon` should be stored in the entity registry as it can be used everywhere (dashboard, automation list). `description` should be stored in the yaml as it's only used in the editor.

I will discuss this point with the rest of the UX/UI team so we can discuss how we manage the migration if we do changes (if it's needed).